Holovibes/sources/compute/analysis.cc

```diff
@@ -194,6 +196,12 @@ void Analysis::init()
     constexpr uint batch_size = 1; // since only one frame.
     // We compute the FFT of the kernel, once, here, instead of every time the
     // convolution subprocess is called
+
+    size_t_buffer_1_.resize(frame_res);
+    size_t_buffer_2_.resize(frame_res);
+    size_t_buffer_3_.resize(frame_res);
+    size_t_gpu_.resize(1);
```

Comment:
Check that the resize function do not raise an error :
err += !size_t_buffer_1_.resize(frame_res);

        if (err != 0)
            throw std::exception(cudaGetErrorString(cudaGetLastError()));